annotator/tests/test_reachable_coverage.py

```
@@ -0,0 +1,78 @@
+    path = tmp_path / "roster.json"
+    path.write_text(json.dumps(rows))
+    return Roster.load(path)
+
+
+def _events(pitcher_idx, batter_idx):
+    """One pitch every 10s, players identified by roster index."""
+    return [
+        PitchEvent(game_pk=1, epoch_s=float(i) * 10, pitcher_id=1000 + p,
+                   pitcher_name=f"P{p}", batter_id=1000 + b, batter_name=f"P{b}",
+                   inning=1, description="")
+        for i, (p, b) in enumerate(zip(pitcher_idx, batter_idx))
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)